presto-noop/src/main/java/io/prestosql/plugin/noop/NoOpPlugin.java

```diff
@@ -48,6 +48,9 @@ public String getName()
         @Override
         public Connector create(String catalogName, Map<String, String> config, ConnectorContext context)
         {
+            if (!config.isEmpty()) {
+                throw new IllegalArgumentException("this connector accepts no configuration properties");
```

Comment:
`this connector does not accept configuration properties`?

Or `Unexpected configuration properties: " + config.keySet()`?

Reply:
let's improve when needed